config/default.schema.json

```diff
@@ -490,6 +490,11 @@
 			"type": "boolean",
 			"default": false,
 			"description": "Enabled that every request is logged with the information route and method"
+		},
+		"JWT_AUD": {
+			"type": "string",
+			"default": "https://hpi-schul-cloud.de",
```

Comment:
In the AC for the ticket it is mentioned that the default should be `https://hpi-schul-cloud.org`. Is this right?

Reply:
my mistake .de is right

Reply:
Then the PR looks good to me